tools/kagami/src/swarm.rs

```diff
@@ -232,7 +232,12 @@ enum PrepareConfigurationStrategy {
 }
 
 impl PrepareConfigurationStrategy {
```

Comment:
The word `Strategy` is a pattern of naming where people name their structures after gang-of-four design patterns. To see why this is a bad convention see the image below.
 
![9c00c55e6d36b73d11a5858fd547a250-73305366](https://github.com/hyperledger/iroha/assets/15856657/97fdf52d-8601-4546-b53a-128159e7a06d)

We should attempt to give this a better name.

Reply:
Hmm, the current "strategy" naming in combination with the enum variant names seems ok for me. I tried to use a different naming, and decided this one to be the best.

The image hasn't helped me to understand why "strategy" is bad in this case. Just a thought that it is avoiding the word "strategy" because of its "bad" history.

Do you have some options for how the naming could be different?

Reply:
I do agree that this naming a bit cumbersome, but still well within readable territory, so I wouldn't insist on changing it. As for alternatives, the only thing coming to mind is maybe `GenerateConfig`? And the same could apply to `PrepareDirectoryStrategy`.

Reply:
I am fine with `GenerateConfig`. Or `PrepareConfig`.